File: Python/1672. Richest Customer Wealth.py
# 1672. Richest Customer Wealth
# Level: Easy
# https://leetcode.com/problems/richest-customer-wealth/

class Solution:
  def maximumWealth(self, accounts: List[List[int]]) -> int:
    biggestAmount = -1
    for account in accounts:
      amount = 0
      for n in account:
        amount += n
      biggestAmount = max(biggestAmount, amount)
      
    return biggestAmount

File: PHP/1672. Richest Customer Wealth.php
<?php
// 1672. Richest Customer Wealth
// Level: Easy
// https://leetcode.com/problems/richest-customer-wealth/

class Solution
{
  function maximumWealth($accounts)
  {
    $biggestAmount = -1;

    for ($i = 0; $i < count($accounts); $i++) {
      $amount = 0;
      for ($j = 0; $j < count($accounts[$i]); $j++) {
        $amount += $accounts[$i][$j];
      }
      $biggestAmount = max($biggestAmount, $amount);
    }
    return $biggestAmount;
  }
}
